test/gov/run_agent.sh

```diff
@@ -0,0 +1,28 @@
+source /tmp/exports_list
+EX_IP_GATEWAY=$(docker inspect exchange-api | jq -r '.[].NetworkSettings.Networks.e2edev_test_network.Gateway')
+EX_HOST_PORT=$(docker inspect exchange-api | jq -r '.[].NetworkSettings.Ports."8080/tcp"[].HostPort')
+export HZN_EXCHANGE_URL="http://$EX_IP_GATEWAY:$EX_HOST_PORT/v1"
+export EXCH_APP_HOST="http://$EX_IP_GATEWAY:$EX_HOST_PORT/v1"
+AGENT_PORT=$(docker port "$(cat /proc/self/cgroup | grep "docker" | sed s/\\//\\n/g | tail -1)" | awk -F'/' '{print $1}')
+echo ${AGENT_PORT}
+DEVICE_NUM=$((${AGENT_PORT} - 8512 + 6))
+echo ${DEVICE_NUM}
+export USER=anax1
```

Comment:
Did you really need to export all of these env vars given that your already sourced many of them on line 1?

Reply:
Defer to future issue when the code is changed to hzn register, much of this will change anyway.